pkg/serviceaccount/jwt.go

```diff
@@ -157,6 +159,7 @@ func (j *jwtTokenAuthenticator) AuthenticateToken(token string) (user.Info, bool
 				if (err.Errors & jwt.ValidationErrorSignatureInvalid) != 0 {
 					// Signature error, perhaps one of the other keys will verify the signature
 					// If not, we want to return this error
+					glog.V(4).Infof("Signature error (key %d): %v", i, err)
```

Comment:
signature verification tries keys in order. if you've rotated the signer, the old signer's public key would get bumped to position 2, and it would be useful to know failures were happening against the first and second public key. I think the index matters more than the key content